phpBB/install/startup.php

```diff
@@ -47,6 +47,36 @@ function phpbb_include_updated($path, $phpbb_root_path, $optional = false)
 	}
 }
 
+function installer_msg_handler($errno, $msg_text, $errfile, $errline)
+{
+	switch ($errno)
+	{
+		case E_NOTICE:
+		case E_WARNING:
+		case E_USER_WARNING:
+		case E_USER_NOTICE:
+			$msg = '[phpBB debug] "' . $msg_text . '" in file ' . $errfile . ' on line ' . $errline;
+			throw new \phpbb\exception\runtime_exception($msg);
```

Comment:
for the debug maybe yes. Even if will often trigger another error later.

You can do that?